src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+
+        System.out.println("Certain machine started its work.");

```

Comment:
The message 'Certain machine started its work.' is too generic. Consider changing it to 'Bulldozer started its work.' to specify which machine is starting.